seed/views/v3/uploads.py

```diff
@@ -59,7 +58,7 @@ class UploadViewSet(viewsets.ViewSet):
             ),
             AutoSchemaHelper.form_string_field(
                 name='source_type',
-                required=True,
+                required=False,
```

Comment:
I would've thought that source_type should still be required? Though admittedly, I don't know why we're handling the "missing" case when treating the request as POST by checking for source_type when treating the request as a GET.
`source_type = request.POST.get('source_type', request.GET.get('source_type'))`